### PR TITLE
Use latest keystone-operator API

### PR DESCRIPTION
**Pull request by fultonj**
The external pkg moved to an api sub module [1]. The glance operator imported both and named them keystonev1beta1 and keystone. Remove keystonev1beta1 references. Redefine keystone import so that it uses the latest keystone-operator API like keystonev1beta1 did.

[1] https://github.com/openstack-k8s-operators/keystone-operator/pull/90

### CODE REVIEW

**Comment by fultonj** [comment]
I did initial testing but want to do further testing before we merge. Early input welcome now however.

**Comment by fmount** [comment]
@fultonj the patch looks good, I have a few more things removed in go.sum but maybe it's my local environment which is a bit messed up [1]. Feel free to check.

[1] https://paste.opendev.org/show/bQpewRHlMRdvUUivZz73/

File: controllers/glanceapi_controller.go [hunk 1]
```diff
@@ -370,7 +369,7 @@ func (r *GlanceAPIReconciler) reconcileInit(
 		return ctrl.Result{}, err
 	}
 
-	ksSvcSpec := keystonev1beta1.KeystoneServiceSpec{
+	ksSvcSpec := keystone.KeystoneServiceSpec{
```
Comment (fmount) [inline controllers/glanceapi_controller.go:372]: +1 makes sense

**Review by fmount — APPROVED** [review]

**Comment by fultonj** [comment]
I've tested further and it's still working.

**Comment by fultonj** [comment]
I'm going to change `keystone` to `keystonev1` so that we version the API.

**Review by dprince — APPROVED** [review]

**Pull request merged**